pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -387,6 +419,14 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 			if mir.Location == "" {
 				return &InvalidRegistries{s: "invalid condition: mirror location is unset"}
 			}
+
+			if reg.MirrorByDigestOnly && mir.PullFromMirror != "" {
+				return &InvalidRegistries{s: "mirror usage set for non-mirror registry and per-mirror at the same time."}
```

Comment:
The error message should include the fields that conflict so that users know directly what to look for.

Maybe something like: `cannot set option A for the registry and option B for the mirror`